### PR TITLE
Fix broken unit test for resumable uploads

### PR DESCRIPTION
**Pull request by abhis3**
### Description

Fix a broken unit test for resumable uploads. These have been an issue point for a few recent bugs, patching to improve test coverage.

### CODE REVIEW

File: scripts/storage-emulator-integration/tests.ts [hunk 1]
```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
           });
         });
 
-        // TODO(abehaskins): This test is temporarily disabled due to a credentials issue
-        it.skip("should handle large (resumable) uploads", async () => {
-          await testBucket.upload(largeFilePath, { resumable: true });
+        it.only("should handle large (resumable) uploads", async () => {
```
Comment (tohhsinpei) [inline scripts/storage-emulator-integration/tests.ts:266]: Delete `only`

File: scripts/storage-emulator-integration/tests.ts [hunk 2]
```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
...
+          const fileName = "test_upload.jpg";
+          const uploadURL = await supertest(STORAGE_EMULATOR_HOST)
+            .post(`/upload/storage/v1/b/${storageBucket}/o?name=${fileName}&uploadType=resumable`)
+            .send({})
```
Comment (tohhsinpei) [inline scripts/storage-emulator-integration/tests.ts:270]: Do you need this `send` then?

File: scripts/storage-emulator-integration/tests.ts [hunk 3]
```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
...
+            .set({
+              Authorization: "Bearer owner",
+            })
```
Comment (tohhsinpei) [inline scripts/storage-emulator-integration/tests.ts:273]: Nit: one line

File: scripts/storage-emulator-integration/tests.ts (hunk 1 above)
Comment (tohhsinpei) [reply scripts/storage-emulator-integration/tests.ts:266]: By the way, what does this unit test have to do with large uploads?

File: scripts/storage-emulator-integration/tests.ts (hunk 3 above)
Comment (abhis3) [reply scripts/storage-emulator-integration/tests.ts:273]: Done

File: scripts/storage-emulator-integration/tests.ts (hunk 1 above)
Comment (abhis3) [reply scripts/storage-emulator-integration/tests.ts:266]: Good call, deleted only

File: scripts/storage-emulator-integration/tests.ts (hunk 2 above)
Comment (abhis3) [reply scripts/storage-emulator-integration/tests.ts:270]: Yup, without it we throw a `400 "Bad Request"`

File: scripts/storage-emulator-integration/tests.ts (hunk 1 above)
Comment (abhis3) [reply scripts/storage-emulator-integration/tests.ts:266]: Nothing in particular with large uploads, this is more to test for resumable functionality. I'll remove the reference to a large file to simplify this.

File: scripts/storage-emulator-integration/tests.ts (hunk 3 above)
Comment (abhis3) [reply scripts/storage-emulator-integration/tests.ts:273]: Actually prettier complains if it's not 2 separate lines, reverting to what it was originally

**Pull request merged**